lua/wire/server/wirelib.lua

```diff
@@ -506,8 +505,8 @@ function WireLib.TriggerOutput(ent, oname, value, iter)
 
 	local output = ent.Outputs[oname]
 	if (output) and (value ~= output.Value or output.Type == "ARRAY" or output.Type == "TABLE") then
-		if (output.TriggerLimit <= 0) then return end
```

Comment:
Good find. That's what you get for writing things past midnight. I was writing this thing in frustration of profiler output. I've actually written this before but forgot to pull request or something: https://github.com/Python1320/wire/commit/3eed48de913d98ea08a02a936fa9d0be970aa03b